ft/ft-ops.cc

```diff
@@ -186,9 +186,37 @@ basement nodes, bulk fetch,  and partial fetch:
  */
 
 static toku_mutex_t ft_open_close_lock;
-
-void
-toku_ft_get_status(FT_STATUS s) {
+static toku_instr_key *ft_open_close_lock_mutex_key;
+// FIXME: the instrumentation keys below are defined here even though they
+// belong to other modules, because they are registered here. If desired, they
+// can be moved to their proper modules and registration done there in a
```

Comment:
We already have the separate file for keys declaration(see toku_instrumentaion.h). But the question is where the keys must be defined. It can be a separate file or the file where the keys are used. Innodb uses the latter approach(just grep mysql_pfs_key_t), and, as I understand, we decided to do the same.  The separate function for keys initialization is a good idea, but I don't think it's reasonable to create new *.cc file just for the only function.